pkgs/proton-ge/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  writeScript,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "proton-ge-custom";
+  version = "GE-Proton8-2";
+
+  src = fetchurl {
+    url = "https://github.com/GloriousEggroll/proton-ge-custom/releases/download/${finalAttrs.version}/${finalAttrs.version}.tar.gz";
+    sha256 = "sha256-gof4yL5sHPKXDC4mDfPyBIvPtWxxxVy6gHx58yoTEbQ=";
+  };
+
+  passthru.updateScript = writeScript "update-pytr" ''
```

Comment:
Likely you don't want to name the update script like that, but something like `update-proton-ge`.

I don't know if you got that error from an older version of my flake package (but as you Name pytr, which is a python module i also have added as a package)

Reply:
I think I got the package from your dots, it was linked to me a few months ago when I was new to NixOS.

To be honest, I am thinking of dropping the updatescript as a whole. It's not hard to update one derivation manually every few weeks.